nucypher/cli/actions.py

```diff
@@ -442,6 +442,14 @@ def confirm_enable_restaking(emitter, staking_address: str) -> bool:
     return True
 
 
+def confirm_enable_winding_down(emitter, staking_address: str) -> bool:
```

Comment:
@vzotova One extra thought here: Do we need a more explicit blurb about winding down, here, and something explicit in the docs?

Reply:
Definitely yes, but I need some help here, because it's not easy to describe it properly

Reply:
How about the following...?

```
Over time, as the locked stake duration decreases i.e. `winds down`, you will receive decreasing inflationary rewards. Instead, by disabling `wind down` (default) the locked stake duration can remain constant until you specify that `wind down` should begin. By keeping the locked stake duration constant, it ensures that you will receive maximum inflation compensation. 

If `wind down` was previously disabled, you can enable it at any point and the locked duration will decrease after each period.

For more information see ([docs link]).
```

^ @vzotova , @michwill , @cygnusv , @mswilkison, @arjunhassard  wdyt?

@vzotova , we should file a separate issue for adding a more in-depth explanation to the docs, perhaps including stake wind-down diagrams and examples.

Reply:
Yeah, diagrams are much required to understand.

I think, it's a good explanation, but I still wouldn't necessarily get it from just text

Reply:
Opened #1551

Reply:
@vzotova should we link to the docs in this blurb? - perhaps as part of the changes for #1558.

Reply:
Done 
✔️